packages/path-compare/src/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Comparison function for two values of the same type `T`.
+ *
+ * Can be used with `Array.prototype.sort` and other similar contexts
+ *
+ * @template T The type of the values to compare
+ * @callback CompareFn
+ * @param {T} a First value
+ * @param {T} b Second value
+ * @returns {number} Negative integer if `a < b`; positive integer if `a > b`;
+ * `0` or `NaN` if equal
+ */
+
+const { stringify: q } = JSON;
+
+/**
+ * String comparison function based on UTF-16 code units
+ *
+ * @type {CompareFn<string>}
+ */
+// eslint-disable-next-line no-nested-ternary
+export const stringCompare = (a, b) => (a === b ? 0 : a < b ? -1 : 1);
```

Comment:
Feel free to keep this export since it can be masked by the top-level thunk. That leaves you free to import it in tests.